source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfig.mm

```diff
@@ -49,7 +49,7 @@ - (instancetype)initWithConfig:(std::shared_ptr<HostConfig> const &)config
 + (ACOHostConfigParseResult *)fromJson:(NSString *)payload resourceResolvers:(ACOResourceResolvers *)resolvers
 {
     ACOHostConfigParseResult *result = nil;
-    
```

Comment:
don't think we need extra newlines here

Reply:
(and below)

---
In reply to: [221061871](https://github.com/Microsoft/AdaptiveCards/pull/1955#discussion_r221061871) [](ancestors = 221061871)